models/fenics-cookies-problem/README.md

```diff
@@ -0,0 +1,132 @@
+BasisDegree     | Integer   | 1         | The degree of the piecewise polynomial FE approximation
+advection       | Integer   | 0         | A flag to add an an advection field to the problem. If set to 1 an advection term with advection field (see below) is added to the PDE problem.
+quad_degree     | Integer   | 8         | The quadrature degree used to evaluate integrals in the matrix assembly.
+diffzero        | Float     | 1.0       | Defines the background diffusion field $$a_0$$ (see below)
+directsolver    | Integer   | 1         | Uses a direct LU solve for linear system. Set to 0 to use GM-RES.
+pc              | String    | 'none'    | Preconditioning for the GM-RES solver, can be either 'ILU' or 'JACOBI'
+tol             | Float     | 1e-4      | Relative tolerance for the GM-RES solver.
+
+### forwardparabolic
+Mapping | Dimensions	| Description
+--------|-------------|------------
+input |	[8] |	These values modify the conductivity coefficient in the 8 cookies, each of them must be greater than -1 (software does not check that input values are valid)
+output |	[1] |	The integral of the solution over the central subdomain $F$ (see below for details)
+
+Feature	| Supported
+--------|---------
+Evaluate|	True
+Gradient|	False
+ApplyJacobian|	False
+ApplyHessian|	False
+
+
+Config          | Type      | Default      | Description
+--------------- |---------- | ----------| -------------
+ N              | integer   | 400       | The number of cells in each dimension (i.e. a mesh of $N^2$ elements).
+ Fidelity       | integer   | 4         | If provided, the mesh will be generated with N = 100 x Fidelity.  The 'fidelity' config key takes precedent over 'N' if both defined.
+BasisDegree     | Integer   | 1         | The degree of the piecewise polynomial FE approximation
+advection       | Integer   | 0         | A flag to add an an advection field to the problem. If set to 1 an advection term with advection field (see below) is added to the PDE problem.
+quad_degree     | Integer   | 8         | The quadrature degree used to evaluate integrals in the matrix assembly.
+diffzero        | Float     | 1.0       | Defines the background diffusion field $$a_0$$ (see below)
+letol           | Float     | 1e-4     | Local timestepping error tolerance for a simple implementation of TR-AB2 timestepping.
+T               | Float      | 10.0     | Final time for timestepping approximation. The QoI is evaluated and returned for time T.
+
+
+
+## Mount directories
+Mount directory | Purpose
+---             |---
+None            | 
+
+## Source code
+
+[Model sources here.](https://github.com/UM-Bridge/benchmarks/tree/main/models/fenics-cookies-problem)
+
+## Description
+
+![cookies-problem](https://raw.githubusercontent.com/UM-Bridge/benchmarks/main/models/cookies-problem/cookies_domain.png "geometry of the cookies problem")
```

Comment:
path needs update for fenics

Reply:
yes, let us know if you want us to take care of it

Reply:
could you make the path changes? I'll handle CI and any formatting stuff after merging